.github/workflows/build_and_test.yml

```diff
@@ -97,8 +97,11 @@ jobs:
 
       - run: docker images --all
 
+      - name: Cargo test in worker
+        run: docker run -rm integritee-worker-${{ env.IMAGE_SUFFIX }} cargo-test --release
```

Comment:
Ah, I see running this, separated, will remove the possibility to benefit from the build cache from before, so maybe it is a bad idea.

Reply:
This is wrong, and could never work from beginn with.